homeassistant/components/tradfri/light.py

```diff
@@ -266,8 +266,6 @@ def hs_color(self):
             brightness = kwargs[ATTR_BRIGHTNESS]
             if brightness > 254:
                 brightness = 254
-            elif brightness < 0:
```

Comment:
This can not happen since brightness is clamped between 0..255 by the light schema, and `async_turn_on` will only be called if brightness is larger than 0 with this PR